cmd/stayrtr/stayrtr.go

```diff
@@ -533,12 +533,21 @@ func (s *state) updateSlurm(file string) (bool, error) {
 		RefreshStatusCode.WithLabelValues(file, fmt.Sprintf("%d", code)).Inc()
 	}
 
+	hsum := newSHA256(data)
+	if s.lasthashCache != nil {
```

Comment:
This is the wrong hash you check here. Should be `lasthashSlurm`.
Also in the `bytes.Compare()` below.